src/main/java/edu/rice/habanero/benchmarks/BenchmarkRunner.java

```diff
@@ -45,13 +41,20 @@ public static void runBenchmark(final String[] args, final Benchmark benchmark)
             System.exit(1);
         }
 
-        final String benchmarkName = benchmark.name();
-        if (benchmarkName.contains("Akka")) {
+        if (benchmark.name().contains("Akka")) {
             AkkaActorState.initialize();
         }
 
+        if (no_evaluation) {
```

Comment:
Instead of this `if`-block we should guard the console output statements in this method with `if (!simpleExecutionRequested) { ... }`